node/.gcloudignore

```diff
@@ -0,0 +1,17 @@
+# This file specifies files that are *not* uploaded to Google Cloud Platform
```

Comment:
Duplicates?

FIne with having top-level or lang-level ignore files for git & cloud, but point would be to avoid having to duplicate for each sample. Not sure why you're adding the same file twice.

Reply:
2nd thought, let's keep this one but I removed the one in avatar-bot/